scripts/custom_lint_checks.py

```diff
@@ -623,6 +618,58 @@ def _add_raise_message(self, missing_excs, node):
             node=node)
 
 
+class ImportOnlyModulesChecker(checkers.BaseChecker):
+    """Checker for import from statements. It checks whether
```

Comment:
`Checker for import-from statements. It checks that modules are only imported.`

Reply:
Done.